go.mod

```diff
@@ -6,7 +6,8 @@ toolchain go1.22.7
 
 require (
 	github.com/Layr-Labs/eigenda v0.8.6-rc.0.0.20250130173836-0d293cc03198
-	github.com/Layr-Labs/eigenda-proxy/client v0.0.0-00010101000000-000000000000
+	github.com/Layr-Labs/eigenda-proxy/client v1.0.0
```

Comment:
do we still need this?

Reply:
nope

Reply:
[acfe191](https://github.com/Layr-Labs/eigenda-proxy/pull/301/commits/acfe191eaec70b2207edb99e34b13e628f8cbb8d)